packages/components/src/internal/url/URLResolver.ts

```diff
@@ -539,6 +539,20 @@ export const STORAGE_BOX_MAPPER = new ActionMapper('query', 'detailsQueryRow', r
     return undefined;
 });
 
+// plate-isPlated.api?sampleId=100
+// map to /rd/samples/100/plates
+export const IS_PLATED_MAPPER = new ActionMapper('plate', 'IsPlated', row => {
```

Comment:
Move this to `@labkey/premium` into `PlateUrls.ts` and add to the `PLATE_MAPPERS` constant.